README.md

```diff
@@ -24,34 +24,50 @@ Use this tool to:
 > - **[Vectara Answer](https://github.com/vectara/vectara-answer)**: Configure a sample Docker app for searching Vectara data.
 > - **[Vectara Ingest](https://github.com/vectara/vectara-ingest)**: Crawl data sources and ingest the data into Vectara.
 
+## How to use
+
+### 1. Install Node and NPM
+
+Create-UI expects you to have [Node and NPM](https://nodejs.org/en/download) installed on your system.
+
+### 2. Generate a codebase
+
+```
+npx @vectara/create-ui
+```
+
+Follow the prompts in the command line to generate a React codebase. You can choose to use our sample data, which is ideal for folks who don't have a Vectara account yet. If you already have data in Vectara you can configure the UI to connect to it. [Learn more below.](#set-up-your-data-in-vectara)
```

Comment:
When I looked at the README, I didn't immediately see the "Learn more below" link. What do you think of:

If you already have data in Vectara [**you can configure the UI to connect to it**](#set-up-your-data-in-vectara).

The longer link text and bold makes it more noticeable, and already gives context about the link content.